packages/apputils/src/instancetracker.ts

```diff
@@ -339,6 +339,10 @@ export class InstanceTracker<T extends Widget>
    * respective widgets.
    */
   async restore(options: InstanceTracker.IRestoreOptions<T>): Promise<any> {
+    if (this._hasRestored) {
+      throw Error('Instance tracker has already restored');
```

Comment:
`throw Error` => `throw new Error`

Reply:
Fixed. I was under the impression they were equivalent in this case.

Reply:
> When `Error` is called as a function rather than as a constructor, it creates and initialises a new Error object. Thus the function call `Error(…)` is equivalent to the object creation expression new Error(…) with the same arguments.
http://es5.github.io/#x15.11.1

You are correct. I didn't know that.

Reply:
Javascript!